src/util/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from "react";
+import constants from "../constants";
+
+import React from "react";
+
+const getLocalitems = () => {
+  let list = localStorage.getItem("lists");
+  console.log(list);
+
+  if (list) {
+    return JSON.parse(localStorage.getItem("lists"));
+  } else {
+    return [];
+  }
+};
+
+useEffect(() => {
```

Comment:
useEffect nhi ayega. we just need to create a generic JAVASCRIPT function.
Study about functions in javascript.